ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -1687,6 +1687,22 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-ui
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn
+- as: baremetalds-ipi-ovn-ipv4-fips-f14-wrs
```

Comment:
```suggestion
- as: baremetalds-ipi-ovn-ipv4-fips-wrs-f14
```

Reply:
I think `wrs` is more like a new test type. It's not associated to the workflow/profile names.